### PR TITLE
Add links to get forum badges at the end of the GSG sections

### PR DESCRIPTION
**Pull request by anthonylusardi-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/getting-started/first-feature.rst [hunk 1]
```diff
@@ -224,11 +224,12 @@ You'll notice that new messages appear in the UI as soon as they are sent (due t
       :scale: 50 %
       :alt: In the create-daml-app as soon as you send the message it will appear in the "Messsaging" panel
 
+.. tip:: You completed the second part of the Getting Started Guide! `Join our forum <https://discuss.daml.com>`_ and share a screenshot of your accomplishment to `get your second of 3 badges <https://discuss.daml.com/badges/126/hey-look-what-i-can-do>`_! Get the third badge by `deploying to project:DABL <https://daml.com/learn/getting-started/deploy-to-dabl/>`_
+
```
Comment (nemanja-da) [inline docs/source/getting-started/first-feature.rst:228]: Third badge is missing but I'm pretty sure you're working on it 😄

**Review by nemanja-da — APPROVED** [review]
ty

File: docs/source/getting-started/first-feature.rst (hunk 1 above)
Comment (anthonylusardi-da) [reply docs/source/getting-started/first-feature.rst:228]: You can only get the third badge with daml.com/learn so I linked to the task where you can get it.

Comment (nemanja-da) [reply docs/source/getting-started/first-feature.rst:228]: Ah, I see. Makes sense.

**Pull request merged**